CONTRIBUTING.md

```diff
@@ -43,6 +43,8 @@ If you have a question about Mongoose (not a bug report) please post it to eithe
   - any mocha flags can be specified with `-- <mocha flags here>`
   - For example, you can use `npm test -- -R spec` to use the spec reporter, rather than the dot reporter (by default, the test output looks like a bunch of dots)
   - execute `npm run test-tsd` to run the typescript tests
+  - execute `npm run ts-benchmark` to run the typescript benchmark "performance test" for one single time.
+  - execute `npm run ts-benchmark-watch` to run the typescript benchmark "performance test" while watching changes on types folder. Note: Make sure to commit all changes before executing this command.
```

Comment:
It's not necessary actually, but I think it's better to do that.
When providing a branch to compare with, the tool will check out that branch "master" and benchmark it, then it will check out the previous branch.
If there were uncommitted changes, It will create a temp commit before checking out "master" and then undo that commit when checking out back.
Then all changes will be "unstaged "even if changes were "staged" before creating the temp commit.